### PR TITLE
Functional tests - Add test 'Pagination of orders table'

### PR DESCRIPTION
**Pull request by nesrineabdmouleh**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | develop
| Description?      | Add test 'Pagination of orders table'
| Type?             | refacto
| Category?         | TE
| BC breaks?        | no
| Deprecations?     | no
| Fixed ticket?     | no
| Related PRs       | no
| How to test?      | no need
| Possible impacts? | no

### CODE REVIEW

**Review by matthieu-rolland — DISMISSED** [review]

**Review by Progi1984 — CHANGES_REQUESTED** [review]
@nesrineabdmouleh Could you remove `.github/workflows/myTesting.yml`, please ?

File: tests/UI/pages/BO/orders/index.js [hunk 1]
```diff
@@ -222,18 +233,31 @@ class Order extends BOBasePage {
     };
   }
 
+  /**
+   * Get number of orders in page
+   * @param page {Page} Browser tab
+   * @returns {Promise<*>}
```
Comment (Progi1984) [inline tests/UI/pages/BO/orders/index.js]:
```suggestion
   * @returns {Promise<number>}
```

**Review by Progi1984 — COMMENTED** [review]
@nesrineabdmouleh A small feedback

File: tests/UI/pages/BO/orders/index.js (hunk 1 above)
Comment (nesrineabdmouleh) [reply tests/UI/pages/BO/orders/index.js]: Done thx :+1:

**Review by Progi1984 — APPROVED** [review]

**Review by SD1982 — APPROVED** [review]

**Comment by Progi1984** [comment]
Thanks @nesrineabdmouleh

**Pull request merged**